### PR TITLE
Split glibc_full in multiple bundles

### PR DESCRIPTION
**Pull request by bauen1**
The new bundles behaves exactly the same as the old glibc_full, dropbear and dhcp have been adapted to use the new bundles and tested.

In theory `glibc_libc` and `glibc_ld-linux` could be replaced by "empty" bundles, since libc and ld-linux are already present in the initramfs.

Feel free to squash the commits, currently there is atleast one commit per library, which might be a bit much.

Closes #117

### CODE REVIEW

File: src/minimal_overlay/bundles/glibc_ld-linux/bundle.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+mkdir -p "$WORK_DIR/overlay/$BUNDLE_NAME"
+cd $WORK_DIR/overlay/$BUNDLE_NAME
+
+DESTDIR="$PWD/${BUNDLE_NAME}_installed"
+
+rm -rf $DESTDIR
+
+mkdir -p $DESTDIR/lib
+cp $SYSROOT/lib/ld-linux.so.2 $DESTDIR/lib/
```
Comment (ivandavidov) [inline src/minimal_overlay/bundles/glibc_ld-linux/bundle.sh]: On 64-bit host environment the generated dynamic loader has the name ``ld-linux-x86-64.so.2`` and the copy operation will fail. This is why I use more generic approach here:

https://github.com/ivandavidov/minimal/blob/9dc8fc226142f1f1cd56cb24ee1adfa94e678b70/src/09_generate_rootfs.sh#L48

Either the whole bundle should be removed (no need to overwrite the dynamic loader) or the copy operation should be enhanced a little in order to handle both 32-bit and 64-bit loaders.

**Comment by ivandavidov** [comment]
My apologies for the ``ln -s`` comment from earlier. You were right - when you create link which points to file in the same folder the actual reference is stripped down to the file itself, without the full path.

One more comment - you can also add meta-package for ``libm`` which is provided by the initramfs.. This library is mandatory or otherwise the BusyBox binary cannot be executed. The library is provided here:

https://github.com/ivandavidov/minimal/blob/9dc8fc226142f1f1cd56cb24ee1adfa94e678b70/src/09_generate_rootfs.sh#L58

**Comment by bauen1** [comment]
I'm not removing them, since some packages don't depend on libc, and statically linking busybox could remove libc from the initramfs.

**Comment by ivandavidov** [comment]
Looks good. I'm merging it.

**Pull request merged**